### PR TITLE
feeding adc1 with random value for initRandomSeed()

### PR DESCRIPTION
**Pull request by dlancer**

### CODE REVIEW

File: src/arduboy_avr.c [hunk 1]
```diff
@@ -133,6 +134,13 @@ void arduboy_avr_button_event(enum button_e btn_e, bool pressed)
 	}
 }
 
+void arduboy_adc_update_hook(struct avr_irq_t *irq, uint32_t value, void *param)
+{
+	avr_irq_t *iop_irq = avr_io_getirq(mod_s.avr, AVR_IOCTL_ADC_GETIRQ, ADC_IRQ_ADC1);
+	u_int16_t milivolts = (u_int16_t)(rand() % ADC_VREF_V256);
```
Comment (dxxb) [inline src/arduboy_avr.c]: Please use standard integer types i.e. `uint16_t` not `u_int16_t`

File: src/arduboy_avr.c [hunk 2]
```diff
@@ -210,6 +219,13 @@ int arduboy_avr_setup(struct sim_arduboy_opts *opts)
 	avr_cycle_timer_register_usec(avr, SSD1306_FRAME_PERIOD_US, update_luma, &mod_s.ssd1306);
 	avr_cycle_timer_register_usec(avr, GL_FRAME_PERIOD_US, render_timer_callback, &mod_s.ssd1306);
 
+	/* Setup initial random seed */
+	srand((u_int)time(NULL));
```
Comment (dxxb) [inline src/arduboy_avr.c]: Ditto

**Review by dxxb — CHANGES_REQUESTED** [review]
Nice! Thank you for the contribution. Please see my comments.

File: src/arduboy_avr.c (hunk 1 above)
Comment (dlancer) [reply src/arduboy_avr.c]: fixed

File: src/arduboy_avr.c (hunk 2 above)
Comment (dlancer) [reply src/arduboy_avr.c]: fixed

File: src/arduboy_avr.c [hunk 3]
```diff
@@ -210,6 +219,13 @@ int arduboy_avr_setup(struct sim_arduboy_opts *opts)
 	avr_cycle_timer_register_usec(avr, SSD1306_FRAME_PERIOD_US, update_luma, &mod_s.ssd1306);
 	avr_cycle_timer_register_usec(avr, GL_FRAME_PERIOD_US, render_timer_callback, &mod_s.ssd1306);
 
+	/* Setup initial random seed */
+	srand((uint)time(NULL));
```
Comment (dxxb) [inline src/arduboy_avr.c]: `uint` is still non-standard, this should be `unsigned int`.

**Review by dxxb — APPROVED** [review]

**Pull request merged**